src/python/build.sh

```diff
@@ -138,7 +159,7 @@ setup_and_run() {
 
             if [ $? -gt 0 ]
             then
-                pip install virtualenv
+                pip install -r requirements-virtualenv.txt
```

Comment:
what else we are expecting in requirements-virtualenv.txt; if nothing.. I believe we should just avoid it.